build/Common.prod.props

```diff
@@ -6,9 +6,6 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftCodeAnalysisAnalyzersPkgVer)" Condition=" $(OS) == 'Windows_NT'">
```

Comment:
if u do this, u are going to remove for all projects. is that the intention?

Reply:
For other projects, PRs are going against main branch. This PR is against metrics branch only, so we shouldn't lose any coverage

Reply:
i see. ok then!